themes/bootstrap3/js/facets.js

```diff
@@ -1,5 +1,87 @@
 /*global VuFind */
 /*exported initFacetTree */
+
+/* --- Facet List --- */
+VuFind.register('facetList', function FacetList() {
+  function overrideHref(selector, overrideParams = {}) {
+    $(selector).each(function overrideHrefEach() {
+      let dummyDomain = 'https://www.example.org'; // we need this since the URL class cannot parse relative URLs
+      let url = new URL(dummyDomain + VuFind.path + $(this).attr('href'));
+      Object.entries(overrideParams).forEach(([key, value]) => {
+        url.searchParams.set(key, value);
+      });
+      url = url.href;
+      url = url.replaceAll(dummyDomain, '');
+      $(this).attr('href', url);
+    });
```

Comment:
Would it be clearer here to use [UrlSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) directly and eliminate the whole dummy domain trick?

```JS
let url = VuFind.path + this.getAttribute("href") + "&" + params.toString();
```

Reply:
If memory serves, based on previous discussion with @mtrojan-ub about this issue, we also need to account for the hash portion of the URL, and UrlSearchParams can't do that.